testing/sdk_marathon.py

```diff
@@ -225,22 +231,29 @@ def _update():
 
 def destroy_app(app_name: str, timeout=TIMEOUT_SECONDS) -> None:
     @retrying.retry(stop_max_delay=timeout * 1000, wait_fixed=2000)
-    def _destroy():
+    def _destroy() -> dict:
         response = sdk_cmd.cluster_request(
             "DELETE", _api_url("apps/{}".format(app_name)), params={"force": "true"}
         )
-        return _handle_marathon_deployment_response(response)
+        return response.json()
 
     result = _destroy()
-    result.raise_on_error()
+    deployment_id = result["deploymentId"]
```

Comment:
How does not raising on error here change the behaviour?

Reply:
I would assume if the `deploymentId` is in the response the `DELETE` was successful otherwise it fails but maybe it's helpful to log the response message here.

Reply:
OK. I see what this is -- this is `_deployment_result.raise_on_error()` which is different to `response.raise_for_status()` which is what I was thinking of. We could follow-up to simplify things a little here, but I don't think it's too critical.

Reply:
This refers probably to the complete way how we handle the marathon response. An improvement here might include looking into other projects handling the parsing the responses correctly.